Validation/MtdValidation/plugins/EtlDigiHitsValidation.cc

```diff
@@ -0,0 +1,255 @@
+  meHitY_[1]      = ibook.book1D("EtlHitYZpos", "ETL DIGI hits Y (+Z);Y_{DIGI} [cm]", 100, -130., 130.);
+  meHitZ_[0]      = ibook.book1D("EtlHitZZneg", "ETL DIGI hits Z (-Z);Z_{DIGI} [cm]", 100, -304.2, -303.4);
+  meHitZ_[1]      = ibook.book1D("EtlHitZZpos", "ETL DIGI hits Z (+Z);Z_{DIGI} [cm]", 100,  303.4,  304.2);
+
+  meHitPhi_[0]    = ibook.book1D("EtlHitPhiZneg", "ETL DIGI hits #phi (-Z);#phi_{DIGI} [rad]", 100, -3.15, 3.15);
+  meHitPhi_[1]    = ibook.book1D("EtlHitPhiZpos", "ETL DIGI hits #phi (+Z);#phi_{DIGI} [rad]", 100, -3.15, 3.15);
+  meHitEta_[0]    = ibook.book1D("EtlHitEtaZneg", "ETL DIGI hits #eta (-Z);#eta_{DIGI}", 100, -2.96, -1.56);
+  meHitEta_[1]    = ibook.book1D("EtlHitEtaZpos", "ETL DIGI hits #eta (+Z);#eta_{DIGI}", 100,  1.56,  2.96);
+
+
+  meHitTvsQ_[0]   = ibook.bookProfile("EtlHitTvsQZneg", "ETL DIGI ToA vs charge (-Z);Q_{DIGI} [ADC counts];ToA_{DIGI} [TDC counts]",
+				      50, 0., 256., 0., 1024.);
+  meHitTvsQ_[1]   = ibook.bookProfile("EtlHitTvsQZpos", "ETL DIGI ToA vs charge (+Z);Q_{DIGI} [ADC counts];ToA_{DIGI} [TDC counts]",
+				      50, 0., 256., 0., 1024.);
+  meHitQvsPhi_[0] = ibook.bookProfile("EtlHitQvsPhiZneg", "ETL DIGI charge vs #phi (-Z);#phi_{DIGI} [rad];Q_{DIGI} [ADC counts]",
+				      50, -3.15, 3.15, 0., 1024.);
+  meHitQvsPhi_[1] = ibook.bookProfile("EtlHitQvsPhiZpos", "ETL DIGI charge vs #phi (+Z);#phi_{DIGI} [rad];Q_{DIGI} [ADC counts]",
+				      50, -3.15, 3.15, 0., 1024.);
+  meHitQvsEta_[0] = ibook.bookProfile("EtlHitQvsEtaZneg","ETL DIGI charge vs #eta (-Z);#eta_{DIGI};Q_{DIGI} [ADC counts]",
+				      50, -2.96, -1.56, 0., 1024.);
+  meHitQvsEta_[1] = ibook.bookProfile("EtlHitQvsEtaZpos","ETL DIGI charge vs #eta (+Z);#eta_{DIGI};Q_{DIGI} [ADC counts]",
+				      50, 1.56,  2.96, 0., 1024.);
+  meHitTvsPhi_[0] = ibook.bookProfile("EtlHitTvsPhiZneg", "ETL DIGI ToA vs #phi (-Z);#phi_{DIGI} [rad];ToA_{DIGI} [TDC counts]",
+				      50, -3.15, 3.15, 0., 1024.);
+  meHitTvsPhi_[1] = ibook.bookProfile("EtlHitTvsPhiZpos", "ETL DIGI ToA vs #phi (+Z);#phi_{DIGI} [rad];ToA_{DIGI} [TDC counts]",
+				      50, -3.15, 3.15, 0., 1024.);
+  meHitTvsEta_[0] = ibook.bookProfile("EtlHitTvsEtaZneg","ETL DIGI ToA vs #eta (-Z);#eta_{DIGI};ToA_{DIGI} [TDC counts]",
+				      50, -2.96, -1.56, 0., 1024.);
+  meHitTvsEta_[1] = ibook.bookProfile("EtlHitTvsEtaZpos","ETL DIGI ToA vs #eta (+Z);#eta_{DIGI};ToA_{DIGI} [TDC counts]",
+				      50, 1.56,  2.96, 0., 1024.);
+
+}
+
+
+// ------------ method fills 'descriptions' with the allowed parameters for the module  ------------
+void EtlDigiHitsValidation::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+
+  edm::ParameterSetDescription desc;
+
+  desc.add<std::string>("folder", "MTD/ETL/DigiHits");
+  desc.add<std::string>("moduleLabel","mix");
+  desc.add<std::string>("etlDigiHitsCollection","FTLEndcap");
+
+  descriptions.add("etlDigiHits", desc);
```

Comment:
To anticipate premixing (#26332), please change this to
```suggestion
  descriptions.add("etlDigiHitsDefault", desc);
```
and create `Validation/MtdValidation/python/btlDigiHits_cfi.py` with
```python
import FWCore.ParameterSet.Config as cms
from Validation.MtdValidation.etlDigiHitsDefault_cfi import etlDigiHitsDefault as _etlDigiHitsDefault
etlDigiHits = _etlDigiHitsDefault.clone()

from Configuration.ProcessModifiers.premix_stage2_cff import premix_stage2
premix_stage2.toModify(etlDigiHits, moduleLabel = "mixData")
```